### DISCUSSION TITLE
Godot 3.x : In the inspector's resource dropdowns, move these options (Quickload, Load, Edit, Clear, Make Unique, Copy, Paste etc) to the top.

### DISCUSSION THREAD

**Discussion by AnnanDistance** [godotengine/godot-proposals#10126] [Editor]
![ResourceMenu](https://github.com/godotengine/godot-proposals/assets/113605192/1101c936-547d-4886-a8b6-61f79ac38aba)
When working with Custom Resources, the long list of options to make new resources is a pain to scroll through whenever we need to perform common actions such as "Copy" or "Make Unique".
Since the feature of custom resource export variable hint is not there in the 3.x branch, it would be great if you could shift these options to the top of the menu.
Thank you.

**Comment by Calinou** [comment]
- See also https://github.com/godotengine/godot/pull/91540.

I recall this being discussed in another proposal or PR, but I can't find it right now.

**Comment by tsikavo** [comment]
It would be nice to have a "new" button in this menu that would open a submenu with all the top items. They really take up a lot of space in certain cases.

**Reply by Calinou** [reply]
> It would be nice to have a "new" button in this menu that would open a submenu with all the top items. They really take up a lot of space in certain cases.

For dropdowns that only have a few options, this would require either an additional click or waiting for a submenu to appear, which would hinder productivity more than it helps. This makes sense for dropdowns that have 20+ options though.

That said, this issue is largely fixed in 4.x thanks to custom resources being able to be used as type hints. Spending a lot of effort on fixing the issue in 3.x may not be worth it in terms of development time. Contributions are welcome either way, but remember that 3.6 will likely enter RC state soon.

**Reply by AnnanDistance** [reply]
I don't think we really need new buttons or submenus. Just have the management options on top of the create options for non-empty slots, if disrupting the existing behavior for creating a new resource in an empty slot is a concern.
